FLEDGE.md

```diff
@@ -299,7 +299,7 @@ The arguments to `generateBid()` are:
       'prevWins': [[time1,ad1],[time2,ad2],...],
     }
     ```
-* sameSiteInterestGroups: if the user is added to `interestGroup` on a single site, this argement contains a list of all other interest group objects added by that site; if the user is added to `interestGroup` on mutliple sites, this argument will be empty.
+* sameSiteInterestGroups: if the user is added to `interestGroup` on a single site, this argument contains a list of all other interest group objects added by that site; if the user is added to `interestGroup` on mutliple sites, this argument will be a empty list.
```

Comment:
nit/typos:
- Capital I for the first "if"
- added by that site -> added on that site
- Instead of the whole phrase "if the user ....", maybe just say "otherwise"?
- mutliple -> multiple
- a empty list -> an empty list

Reply:
fixed, thanks.